api-server/src/graphql/schema.graphql

```diff
@@ -394,6 +391,17 @@ type JobAd {
 	company: Company!
 }
 
+type JobApplication {
```

Comment:
Should we return this on the public API? This is private information. It at least needs a permissions check.

Reply:
Went ahead and added a companyId check to make sure the right company is querying the data.